Simplenote/src/androidTest/java/com/automattic/simplenote/screenshots/ScreenshotTest.java

```diff
@@ -368,6 +376,11 @@ private void takeInterNoteLinkingScreenshotFromNotesList() throws InterruptedExc
         onView(withText("Trash")).perform(click());
         onView(withId(R.id.menu_empty_trash)).perform(click());
         onView(withText("Empty")).perform(click());
+
+        // Go back to notes list
+        loadSideMenuFromNotesList();    // The trash activity is obviously not the notes list one,
+                                        // but this method works anyways
+        onView(withText("All Notes")).perform(click());
```

Comment:
This logic was previously in the main method, but it belongs here because it restores the app to the state in which it was before calling this method.